src/bluehawk/parser/makeAttributesConformToJsonSchemaRule.ts

```diff
@@ -1,12 +1,12 @@
 import { AnyTagNode } from "../parser";
-import Ajv, { JSONSchemaType, AnySchema } from "ajv";
+import Ajv, { AnySchema } from "ajv";
 import { Rule, ValidateCstResult } from "./validator";
 
 const ajv = new Ajv();
 
 // Creates a rule that checks attributes against the given JSON schema.
 export const makeAttributesConformToJsonSchemaRule = <Type = unknown>(
```

Comment:
question: are you still using `Type` at all here anymore, or always unknown? 

if always unknown, can we delete smthn here?